api4/team_test.go

```diff
@@ -493,4 +493,36 @@ func TestGetMyTeamsUnread(t *testing.T) {
 	Client.Logout()
 	_, resp = Client.GetTeamsUnreadForUser(user.Id, "")
 	CheckUnauthorizedStatus(t, resp)
-}
\ No newline at end of file
+}
+
+func TestGetTeamMembersByIds(t *testing.T) {
+	th := Setup().InitBasic()
+	defer TearDown()
+	Client := th.Client
+
+	tm, resp := Client.GetTeamMembersByIds(th.BasicTeam.Id, []string{th.BasicUser.Id})
+	CheckNoError(t, resp)
+
+	if tm[0].UserId != th.BasicUser.Id {
+		t.Fatal("returned wrong user")
+	}
+
+	_, resp = Client.GetTeamMembersByIds(th.BasicTeam.Id, []string{})
+	CheckBadRequestStatus(t, resp)
```

Comment:
Can you add test cases checking bad team ids? One that's totally wrong, one that's `model.NewId()`, etc.